src/pages/about-us.tsx

```diff
@@ -49,25 +49,30 @@ const AboutUsPage = () => {
           </div>
         </StyledMainSection>
 
-        <section className="w-full max-w-1440 ">
+        <section className="w-full">
```

Comment:
The `max-w-1440` is required so it doesn't span the entire screen on monitor displays and larger screen